barcelona.html

```diff
@@ -253,18 +253,21 @@ <h2 class="lg:text-5xl text-2xl font-semibold">uncodebar highlights</h2>
     <section id="sponsor" class="lg:p-20 p-10">
         <div class="flex flex-wrap">
 
+          <div class="md:w-1/4"></div>
+
           <div class="md:w-2/4 flex w-screen flex-col lg:p-20 p-10 items-center">
-            <img class="pb-5 max-h-28" src="images/contentful.png" alt="a logo of contentful">
-            <p class="text-center text-lg max-w-md font-semibold mb-10">More than a headless CMS, Contentful is the
-              API-first composable content platform to create, manage and publish content on any digital channel.</p>
-            <a href="https://www.contentful.com/" class="bg-gradient-to-tr from-button_blue to-button_blue_ py-5 px-9 rounded-full text-white" target="_blank">FIND OUT MORE</a>
+            <img class="pb-5 max-h-28" src="images/travelperk-logo.jpeg" alt="a logo of TravelPerk">
```

Comment:
You've change this image URL but I don't see the travelperk image in the pull request

Reply:
It was already on the repo, so I just used that one and their description text from their website

Reply:
Ah yeah course, because of London 🤦‍♀️